frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/create-local-volume-set.tsx

```diff
@@ -0,0 +1,274 @@
+import { k8sCreate, referenceFor } from '@console/internal/module/k8s';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager';
+import { LocalVolumeSetModel, LocalVolumeModel } from '../../models';
+import { NodesSelectionList } from './nodes-selection-list';
+import { RowUIDMap, LocalVolumeSetKind } from './types';
+import { getSelectedNodeUIDs } from './utils';
+import './create-local-volume-set.scss';
+
+const volumeTypeOptions = Object.freeze({
+  'SSD/NVMe': 'SSD/NVMe',
+  HDD: 'HDD',
+});
+const volumeModeOptions = Object.freeze({
+  Block: 'Block',
+  Filesystem: 'Filesystem',
+});
+const volumeSizeUnitOptions = Object.freeze({
+  TiB: 'TiB',
+  GiB: 'GiB',
+});
+
+const CreateLocalVolumeSet: React.FC = withHandlePromise((props: CreateLocalVolumeSetProps) => {
+  const { match, handlePromise, inProgress, errorMessage } = props;
+  const [volumeSetName, setVolumeSetName] = React.useState('');
+  const [storageClassName, setStorageClassName] = React.useState('');
+  const [showNodesList, setShowNodesList] = React.useState(false);
+  const [volumeType, setVolumeType] = React.useState(volumeTypeOptions['SSD/NVMe']);
+  const [volumeMode, setVolumeMode] = React.useState(volumeModeOptions.Block);
+  const [minVolumeSize, setMinVolumeSize] = React.useState('0');
+  const [maxVolumeSize, setMaxVolumeSize] = React.useState('');
+  const [volumeSizeUnit, setVolumeSizeUnit] = React.useState(volumeSizeUnitOptions.TiB);
+  const [maxVolumeLimit, setMaxVolumeLimit] = React.useState('');
+  const [rows, setRows] = React.useState<RowUIDMap>({});
+  const [allSelected, setAllSelected] = React.useState(null);
+
+  const { ns, appName } = match.params;
+  const modelName = LocalVolumeSetModel.label;
+
+  const toggleShowNodesList = () => {
+    setShowNodesList(!showNodesList);
+  };
+
+  const onSubmit = (event: React.FormEvent<EventTarget>) => {
+    event.preventDefault();
+    const requestData: LocalVolumeSetKind = {
+      apiVersion: LocalVolumeSetModel.apiVersion,
+      kind: LocalVolumeSetModel.kind,
+      metadata: { name: volumeSetName },
+      spec: {
+        storageClassName,
+        volumeMode,
+      },
+      // @TODO: more fields will be added
+    };
+    if (showNodesList) {
+      const selectedNodesUID = getSelectedNodeUIDs(rows);
+      const selectedNodes = selectedNodesUID.map((uid) => rows[uid].props.data.metadata.name);
+      requestData.spec.nodeSelector = {
+        nodeSelectorTerms: [
+          {
+            matchExpressions: [
+              { key: 'kubernetes.io/hostname', operator: 'In', values: [...selectedNodes] },
+            ],
+          },
+        ],
+      };
+    }
+    handlePromise(k8sCreate(LocalVolumeModel, requestData))
+      .then((resource) => {
+        history.push(resourceObjPath(resource, referenceFor(resource)));
+      })
+      .catch((e) => {
+        // eslint-disable-next-line no-console
+        console.error(e);
```

Comment:
This is pointless if you're already showing the error in the ActionBar.

Reply:
Sure, I think just catch(() -> null) here would be suffice for linter